src/odemis/gui/model/__init__.py

```diff
@@ -1238,7 +1238,7 @@ def __init__(self, name, stage=None, stream_classes=None, fov_hw=None, projectio
         else:
             self.stream_classes = stream_classes
         self._stage = stage
-        
+
```

Comment:
Somehow I cannot change these lines back to the original. Maybe someone added tabs here instead of 4 spaces?